ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx

```diff
@@ -314,6 +317,7 @@ export class ScenePlayerImpl extends React.Component<
       getDurationHook,
       seekHook,
       getCurrentTimeHook,
+      ...(!ScreenUtils.isMobile() && extras),
```

Comment:
IMO it's kinda odd to drop `extras` when on mobile (as it's not clear from the variable name). Personally I would change it to always include `extras` but either make it an empty object on mobile or explicitly enable/disable the floating player.

So either:
```js
const extras = {};
if (!ScreenUtils.isMobile()) {
  extras.floating = {
    dismissable: true,
  };
```

Or just change original and utilize `mode`:
```js
      height: "100%",
      floating: {
        dismissible: true,
        mode: ScreenUtils.isMobile() ? 'never', 'notVisible',
      },
      cast: {},
```

Note that the latter requires JWPlayer 8.17+ and I don't know which version is being used at the moment.